share/spice/renego_job_search/renego_job_search.js

```diff
@@ -0,0 +1,52 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_renego_job_search = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error || !api_result.responseData
+                || !api_result.responseData.jobs || !api_result.responseData.jobs.length) {
+            return Spice.failed('renego_job_search');
+        }
+
+        var responseData = api_result.responseData;
+        var queryParams = responseData.qparams;
+
+        DDG.require('moment.js', function(){
+            // Render the response
+            Spice.add({
+                id: "renego_job_search",
+                name: "Jobs",
+                data: responseData.jobs,
+                meta: {
+                    sourceName: "Renego",
+                    sourceUrl: 'https://www.renego.de/?w=' + encodeURIComponent(queryParams.w) + '&l=' + encodeURIComponent(queryParams.l) + '&cid=duckduckgo&csid=label'
```

Comment:
@mightycid one more small bug: we need to check that `queryParams.l` is defined, otherwise you end up with a search including "null" on Renego:

![javascript_null_jobs_-_renego_de](https://cloud.githubusercontent.com/assets/873785/12343595/43e81d70-bb03-11e5-8578-5349966bd200.png)